src/components/DepositWidget/index.tsx

```diff
@@ -285,6 +286,29 @@ const BalanceTools = styled.div`
   }
 `
 
+const NoTokensMessage = styled.tr`
+  /* increase speicifcity */
+  &&&&&& {
```

Comment:
ewwwwwww! hahaha love it. is there a workaround?

Reply:
`!important`, which I like even less

Reply:
mm no I'd say !important is like TODO and can be changed later easier

Reply:
But this one isn't meant to be changed
It's meant to override otherwise applied styles from `table > td`
And raising specificity is a know technique, whereas `!important` is a css-resetting hack

Reply:
ok